iml-agent/src/daemon_plugins/ostpool.rs

```diff
@@ -10,13 +10,13 @@ use crate::{
 };
 use futures::{future::join_all, lock::Mutex, Future, FutureExt};
 use iml_wire_types::{FsPoolMap, OstPool};
-use std::{collections::BTreeSet, pin::Pin, sync::Arc};
+use std::{collections::BTreeSet, pin::Pin, sync::Arc, time::{Duration, Instant}};
 
 /// Resend full tree every 5 min
-const DEFAULT_RESEND: u32 = 300;
+const DEFAULT_RESEND: u64 = 300;
```

Comment:
Just set the duration directly here instead

```suggestion
const DEFAULT_RESEND: Duration = Duration::from_secs(300);
```